config/workload_all.xml

```diff
@@ -49,6 +49,6 @@
           <weights>45,43,4,4,4</weights>
         </work>
 	</works>
-    <maxRetriesPerTransaction>2</maxRetriesPerTransaction>
+    <maxRetriesPerTransaction>0</maxRetriesPerTransaction>

```

Comment:
can you add a comment here as to why we set this to zero?

Reply:
Done